### DISCUSSION TITLE
Support for Custom trained  llm models from hugging  face

### DISCUSSION THREAD

**Discussion by shrirudragoud** [vllm-project/vllm#2392] [Q&A]
I am Training my own model using the hugging face mistral llm and i want to know how can i use the vllm for my own trained model which i can run on my own onprem server

**Comment by chenxu2048** [comment]
If the architecture of your model remains unchanged during training, it is supported in vLLM.  You can check out all the supported models at [this page](https://docs.vllm.ai/en/latest/models/supported_models.html).

**Reply by mcpark84** [reply]
I've encountered the same issue. Generally, when we fine-tune a model based on what Hugging Face provides and save it, a file in a format that can be loaded by Hugging Face is generated. How can I use such a saved model in vLLM?
